samples/local_attestation/common/attestation.cpp

```diff
@@ -12,154 +19,159 @@ Attestation::Attestation(Crypto* crypto, uint8_t* enclave_mrsigner)
 }
 
 /**
- * Generate a report for the given data. The SHA256 digest of the data is
- * stored in the report_data field of the generated report.
+ * Generate evidence for the given data.
  */
-bool Attestation::generate_local_report(
+bool Attestation::generate_local_attestation_evidence(
     uint8_t* target_info_buffer,
     size_t target_info_size,
     const uint8_t* data,
     const size_t data_size,
-    uint8_t** report_buf,
-    size_t* remote_report_buf_size)
+    uint8_t** evidence_buf,
+    size_t* local_evidence_buf_size)
 {
     bool ret = false;
     uint8_t sha256[32];
-    oe_result_t result = OE_OK;
-    uint8_t* temp_buf = NULL;
+    oe_result_t result = OE_OK;    
+    oe_result_t attester_result = OE_OK;
+    oe_result_t attester_format_result = OE_OK;
 
     if (m_crypto->Sha256(data, data_size, sha256) != 0)
     {
         goto exit;
     }
 
-    // To generate a local report that just needs to be attested by another
-    // enclave running on the same platform, set flags to 0 in oe_get_report
-    // call. This uses the EREPORT instruction to generate this enclave's local
-    // report.
-    result = oe_get_report(
-        0,      // get a local report
-        sha256, // Store sha256 in report_data field
-        sizeof(sha256),
-        target_info_buffer,
-        target_info_size,
-        &temp_buf,
-        remote_report_buf_size);
+    // Initialize attester and use the SGX plugin.
+    attester_result = oe_attester_initialize();
+    if (attester_result != OE_OK)
+    {
+        TRACE_ENCLAVE("oe_attester_initialize failed.");
+        goto exit;
+    }
+
+    // Generate evidence based on the format selected by the attester.
+    result = oe_get_evidence(&sgx_local_uuid, NULL, NULL, 0, target_info_buffer, target_info_size, evidence_buf, local_evidence_buf_size, NULL, 0);
```

Comment:
Have you tried to reformat the code? This line seems to be long.
`./scripts/format-code`

Reply:
Fixed.